src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.SqlClient.cs

```diff
@@ -910,5 +910,17 @@ internal static ArgumentException InvalidMixedArgumentOfSecureCredentialAndInteg
         {
             return Argument(SR.GetString(SR.ADP_InvalidMixedUsageOfSecureCredentialAndIntegratedSecurity));
         }
+        internal static InvalidOperationException InvalidMixedUsageOfAccessTokenAndIntegratedSecurity()
+        {
+            return ADP.InvalidOperation(SR.GetString(SR.ADP_InvalidMixedUsageOfAccessTokenAndIntegratedSecurity));
+        }
+        static internal InvalidOperationException InvalidMixedUsageOfAccessTokenAndUserIDPassword()
+        {
+            return ADP.InvalidOperation(SR.GetString(SR.ADP_InvalidMixedUsageOfAccessTokenAndUserIDPassword));
+        }
+        static internal Exception InvalidMixedUsageOfCredentialAndAccessToken()
```

Comment:
Nit: it'd be good to consistently use "internal static" rather than sometimes using "static internal"

Reply:
done.